include/igl/line_field_mismatch.cpp

```diff
@@ -81,7 +81,7 @@ class MissMatchCalculatorLine
 
 public:
 
-    inline MissMatchCalculatorLine(const Eigen::PlainObjectBase<DerivedV> &_V,
+    inline MismatchCalculatorLine(const Eigen::PlainObjectBase<DerivedV> &_V,
```

Comment:
Same here, should probably start with lowercase.

Reply:
Let me know what is the final decision about the change. I am in favor of the camelCase for methods, as I said before.